static/templates/message_edit_form.hbs

```diff
@@ -8,6 +8,12 @@
     {{#if is_stream}}
     <div class="control-group no-margin">
         <div class="controls edit-controls">
+            <select class="inline_select_topic_edit" id="select_stream_id_{{ message_id }}" {{#unless show_edit_stream}}style="display:none"{{/unless}}>
+                <option value="{{ stream_id }}" selected='selected'>{{ stream_name }}</option>
+                {{#each available_streams}}
+                <option value="{{ this.stream_id }}">#{{this.name}}</option>
+                {{/each}}
+            </select>
             <input type="text" placeholder="{{topic}}" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
```

Comment:
This widget is not vertically aligned with the topic, which looks off:

![image](https://user-images.githubusercontent.com/2746074/81457769-4fc3de80-914c-11ea-8775-2744e14bcf4d.png)

We should also put a `>` between the two textboxes, and add the stream color, to make it more visually clear that this is a stream field.

Reply:
The styling we use in compose:

![image](https://user-images.githubusercontent.com/2746074/81457792-679b6280-914c-11ea-8273-eb026479e2cd.png)

is what I have in mind